optimizely_server_side.gemspec

```diff
@@ -2,7 +2,7 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = 'optimizely_server_side'
-  s.version     = '1.2.2'
+  s.version     = '1.2.3'
```

Comment:
Can you also do `bundle install` so we have latest `Gemfile.lock` with this version. Also, I would prefer having a version `2.0.0` as this adds a new `major` feature and not a `patch`

Reply:
Will do